src/bytes_mut.rs

```diff
@@ -625,7 +652,7 @@ impl BytesMut {
         let original_capacity_repr;
 
         unsafe {
-            original_capacity_repr = (*shared).original_capacity_repr;
+            original_capacity_repr = (*shared).original_capacity_repr;  
```

Comment:
This is probably causing a rustfmt CI failure.

Reply:
Oh, that's an oversight / accidental whitespace modification.

Reply:
I squashed the two commits from today and fixed this whitespace.